.github/workflows/wheels.yaml

```diff
@@ -63,6 +63,11 @@ jobs:
     runs-on: ubuntu-latest
     # upload to PyPI on every tag starting with 'yt_astro_analysis-'
     if: github.event_name == 'push' && startsWith(github.event.ref, 'refs/tags/yt_astro_analysis-')
+    environment:
+      name: pypi
```

Comment:
from the email I got when trusted-publishing was enabled on PyPI:
> Environment: upload_pypi

The same name should be used here (but you could also change it on PyPI if you haven't already)

Reply:
Note that you'll also need to actually create the env on the repo's side